WinUIGallery/Samples/ControlPages/ContentDialogPage.xaml.cs

```diff
@@ -21,28 +21,67 @@ private async void ShowDialog_Click(object sender, RoutedEventArgs e)
 
         // XamlRoot must be set in the case of a ContentDialog running in a Desktop app
         dialog.XamlRoot = this.XamlRoot;
-        dialog.Style = Application.Current.Resources["DefaultContentDialogStyle"] as Style;
         dialog.Title = "Save your work?";
         dialog.PrimaryButtonText = "Save";
         dialog.SecondaryButtonText = "Don't Save";
         dialog.CloseButtonText = "Cancel";
         dialog.DefaultButton = ContentDialogButton.Primary;
         dialog.Content = new ContentDialogContent();
-        dialog.RequestedTheme = (VisualTreeHelper.GetParent(sender as Button) as StackPanel).ActualTheme;
 
         var result = await dialog.ShowAsync();
 
         if (result == ContentDialogResult.Primary)
         {
             DialogResult.Text = "User saved their work";
+            var peer = FrameworkElementAutomationPeer.FromElement(DialogResult) ?? FrameworkElementAutomationPeer.CreatePeerForElement(DialogResult);
```

Comment:
Would it make sense to just have a function that sets the dialogresult.text so we don't need the duplication of this code here?